spark-plugins/src/main/java/co/cask/hydrator/plugin/spark/HashingTFFeatureGenerator.java

```diff
@@ -79,11 +81,12 @@ public void initialize(SparkExecutionPluginContext context) throws Exception {
       return context.getSparkContext().emptyRDD();
     }
     outputSchema = outputSchema != null ? outputSchema : config.getOutputSchema(input.first().getSchema());
-    final Map<String, String> mapping = config.getFeatureListMapping();
+    final Map<String, String> mapping = SparkUtils.getFeatureListMapping(config.outputColumnMapping);
```

Comment:
It is not related to spark though. it is actually just a single line that parses a string into a map, wrapped around some error message.  In general, we want to avoid utility methods wherever possible, as they tend to indicate logic is not in the correct place. I don't see much value in this method, as the error message should be specific to the plugin that uses it.